packages/odyssey-react-mui/src/TagList.tsx

```diff
@@ -30,7 +30,13 @@ const TagList = ({ children }: TagListProps) => {
   );
 
   return (
-    <Stack component="ul" direction="row" spacing={2}>
+    <Stack
```

Comment:
@KevinGhadyani-Okta here's another place where having our own class names accessible would be helpful.

Reply:
What do you mean?

Reply:
@KevinGhadyani-Okta `direction` and `spacing` here are both CSS properties being applied via `sx` -- basically using a `style` attribute rather than via `components.ts` or another style layer. We have the same problem with `Form` and `Fieldset`.

Nothing wrong with this PR, but we should find a solution that makes our styling application consistent across components.